space-concordia-robotics/robotics-prototype
# Migrate robotics orin repo to this repo 622

# Assignee Section

## Description

* Replaced old code with the new stuff from robotics-orin. Up to date with code on master branch in robotics-orin as of May 25th 2024)
* Stuck to the old directory structure for now. (TODO: move gps cpp files to separate directory) 

### Steps for Testing
[Imagine you are the reviewer and did not write the code. Write all the steps necessary to test the feature(s) in an **ordered list**. Be **specific** and include important details here so testers don't need to jump through hoops to test your pull request. Be explicit in all the steps, and make sure the expected output is clearly stated.

1. Launch jetson
2. Go to rover/rospackages
3. build packages
4. Test the following on the rover: a.  wheels functionality(wheels_controller) b. arm control (arm_controller) c. arm ik(arm_ik) 
5. Verify that the behaviour remains unchanged compared to robotics-orin main branch
6. Use ros2 topic echo to monitor logs

closes #622

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
